assets/parameters.toml

```diff
@@ -471,3 +465,395 @@ d3.bj = {s6=0.3223, a1=0.0, s8=0.0, a2=4.5050, doi="10.1021/acs.jpca.1c01295"}
 
 [parameter.dsdpbep86drpa75]
 d3.bj = {s6=0.3012, a1=0.0, s8=0.0, a2=4.5050, doi="10.1021/acs.jpca.1c01295"}
+
+[parameter.dsdpbep86_2011]
+d3.bj = {s6=0.418, a1=0.0, s8=0.0, a2=5.65, doi="10.1039/c1cp22592h"}
+
+# dRPA75-D3(BJ) from MRCC source code
+[parameter.drpa75_mrcc]
+d3.bj = {s6=0.375, a1=0.0, s8=0.0, a2=4.505}
+
+# SCS-dRPA75-D3(BJ) from MRCC source code
+[parameter.scsdrpa75_mrcc]
+d3.bj = {s6=0.375, a1=0.0, s8=0.0, a2=4.505}
```

Comment:
I updated all parameters in this file, going forward any parameter addition will require proper attribution. If this cannot be provided, parameters will not be accepted in DFT-D3.